<?xml version="1.0"?>

<launch>

	<node ns="corner_localization" name="relative_localization" pkg="relative_localization" type="relative_localization_node" output="screen">
		<rosparam command="load" file="$(find relative_localization)/ros/launch/corner_localization_params_RAW.yaml"/>
		<param name="localization_method" value="corner" />
	</node>

</launch>
